OpenTAP.TUI/TuiSettings.cs

```diff
@@ -173,12 +173,21 @@ public int TestPlanGridHeight
 
         [Display("Map", Group: "Key Mapping")]
         public List<KeyMap> KeyMap { get; set; } = new List<KeyMap>();
-        
+
         [Browsable(true)]
         [XmlIgnore]
         [Display("Restore Key map", Group: "Key Mapping")]
         public Action ResetKeyMapping { get; set; }
 
+
+        [Browsable(true)]
+        [Display("Scrollback limit", Group: "Log Panel")]
+        public Enabled<int> LogScrollbackLimit { get; set; } = new Enabled<int>() { IsEnabled = false, Value = 500 };
+
+        [Browsable(true)]
+        [Display("Clear on Run", Group: "Log Panel")]
+        public bool ClearOnRun { get; set; } = true;
```

Comment:
Are we sure this should be the default?

```suggestion
        public bool ClearOnRun { get; set; }
```